packages/fuels-core/src/json_abi.rs

```diff
@@ -211,6 +211,11 @@ impl ABIParser {
             ParamType::Bool => Ok(Token::Bool(trimmed_value.parse::<bool>()?)),
             ParamType::Byte => Ok(Token::Byte(trimmed_value.parse::<u8>()?)),
             ParamType::B256 => {
+                if trimmed_value.len() != 64 {
```

Comment:
Would putting 64 behind a `const`ant provide more cleaner and descriptive code?

Reply:
Thanks for the input @leviathanbeak. I added the constant.